frontend/components/GitHubRepoUrlField/RenderField.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const RenderField = ({
```

Comment:
This is a bit of a weird separation of components in my opinion, but I couldn't figure a better way to setup for testing.

Reply:
Yeah, their may be another name we can use besides `RenderField`, like `GithubUrlInput` or something like that?

Reply:
yeah, I picked `RenderField` since that is the prop it is passed into in the "main" component. Also it's in a sub-directory, so I think keeping it called `RenderField` for this and future similar components might be good for consistency.

Reply:
Cool deal, if you like it then it works for me.